examples/hello.rs

```diff
@@ -69,6 +69,13 @@ impl TabViewer for MyContext {
         tab.as_str().into()
     }
 
+    fn closeable(&mut self, tab: &mut Self::Tab) -> bool {
+        match tab.as_str() {
```

Comment:
You can just return `tab == "Inspector"`

Reply:
I actually did that on purpose to make it fit the other implementations of `TabViewer` in `hello.rs`, but feel free to changer that 😄 

Clippy didn't complain locally, I guess that might be a difference between main and the release branch?

Reply:
Might be that you have a different version of the compiler. In 1.71 they added new Clippy lints :D

Reply:
Ah, I forgot `--all-targets` because. I see it now